contracts/gen.sh

```diff
@@ -0,0 +1,3 @@
+abigen -sol Lottery.sol  -out Lottery.go --pkg contracts
+# abigen -sol Faucet.sol  -out Faucet.go --pkg contracts
+# abigen -sol StakeLockContract.sol  -out StakeLockContract.go --pkg contracts
```

Comment:
Any reason to comment out Faucet and StakeLockContract?

Reply:
if i run gen for Faucet and StateLockContract if it will generate the diff between the current go files and the new ones